src/views/HomeView.vue

```diff
@@ -109,6 +134,11 @@ const sportsWithIcon = computed(() => allSports.value.map(sport => ({
   icon: getIcon(sport.key)
 })))
 
+const linksWithIcon = computed(() => links.value.map( link => ({
+  text: link.text[0].toUpperCase() + link.text.substring(1),
+  icon: getIcon(link.text)
+}
+) ))
```

Comment:
```suggestion
const linksWithIcon = computed(() => links.value.map(link => ({
  text: link.text[0].toUpperCase() + link.text.substring(1),
  icon: getIcon(link.text)
})))
```

Reply:
In the future, passing around components like this really isn't the best pattern to follow.

Instead you'd want to have a component live where it's being called, and probably make a `Icon` component where you would pass these values and dynamically import the component.

But for this case it's fine.